spec/requests/articles/articles_show_spec.rb

```diff
@@ -36,7 +36,7 @@
         "publisher" => {
           "@context" => "http://schema.org",
           "@type" => "Organization",
-          "name" => "#{SiteConfig.community_name} #{SiteConfig.collective_noun}",
+          "name" => SiteConfig.community_name.to_s,
```

Comment:
This seems to be failing on Travis but is passing locally. 🤔

Reply:
Hm, is this still failing? I looked on Travis and I don't see any failures for this file (it looks like it was run in [this job](https://travis-ci.com/github/forem/forem/jobs/457485539) but that build is all green). It passes for me locally too ✅ 

Maybe it was flaky?

Reply:
I'm rerunning the build now, so hopefully it'll pass this time around! I could have totally been looking at an old build, as I had ~4 Tarvis tabs open. 😅